pootle/apps/pootle_store/views.py

```diff
@@ -478,8 +478,10 @@ def get_context_data(self, *args, **kwargs):
             if 'virtualfolder' in settings.INSTALLED_APPS
             else None)
         suggestions = self.object.get_suggestions()
+        latest_target_submission = self.object.get_latest_target_submission()
         return {
             'unit': self.object,
+            'latest_target_submission': latest_target_submission,
```

Comment:
Passing instead `'accepted_suggestion': latest_target_submission.suggestion` will allow to simplify template.